<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prescription_00</class>
 <widget class="QWizardPage" name="Prescription_00">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>570</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="patientBox">
     <property name="title">
      <string notr="true">Patient</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lastLabel">
        <property name="toolTip">
         <string comment="PATIENT LAST DISPLAY TOOLTIP">PATIENT LAST</string>
        </property>
        <property name="text">
         <string notr="true">PH PATIENT LAST</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string notr="true">,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstLabel">
        <property name="toolTip">
         <string comment="PATIENT FIRST DISPLAY TOOLTIP">PATIENT FIRST</string>
        </property>
        <property name="text">
         <string notr="true">PH PATIENT FIRST</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="medicationLabel">
       <property name="toolTip">
        <string comment="MEDICATION/BARCODE SEARCH TOOLTIP">MEDICATION/BARCODE</string>
       </property>
       <property name="text">
        <string comment="MEDICATION/BARCODE SEARCH LABEL">MEDICATION/BARCODE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="medicationField">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="MEDICATION/BARCODE SEARCH TOOLTIP">MEDICATION/BARCODE</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string notr="true">Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DisplayTable" name="resultTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">shipments.id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="MEDICATION DISPLAY LABEL">MEDICATION</string>
      </property>
      <property name="toolTip">
       <string comment="MEDICATION DISPLAY TOOLTIP">MEDICATION</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="LOT DISPLAY LABEL">LOT</string>
      </property>
      <property name="toolTip">
       <string comment="LOT DISPLAY TOOLTIP">LOT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="FORM DISPLAY LABEL">FORM</string>
      </property>
      <property name="toolTip">
       <string comment="FORM DISPLAY TOOLTIP">FORM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="STRENGTH DISPLAY LABEL">STRENGTH</string>
      </property>
      <property name="toolTip">
       <string comment="STRENGTH DISPLAY TOOLTIP">STRENGTH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="UNIT SIZE DISPLAY LABEL">UNIT SIZE</string>
      </property>
      <property name="toolTip">
       <string comment="UNIT SIZE DISPLAY TOOLTIP">UNIT SIZE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="PRODUCT LEFT DISPLAY LABEL">PRODUCT LEFT</string>
      </property>
      <property name="toolTip">
       <string comment="PRODUCT LEFT DISPLAY TOOLTIP">PRODUCT LEFT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="EXPIRATION DISPLAY LABEL">EXPIRATION</string>
      </property>
      <property name="toolTip">
       <string comment="EXPIRATION DISPLAY TOOLTIP">EXPIRATION</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayTable</class>
   <extends>QTableWidget</extends>
   <header>displaytable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>medicationField</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>medicationField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
